src/Common/src/Common/Options/FilePathInOptionsChangeTokenSource.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+
+namespace Steeltoe.Common.Options;
```

Comment:
This is bad practice, as the namespace clashes with the built-in `Options` type.

Reply:
I propose `Steeltoe.Common.Configuration`.